generator/src/generate-records.js

```diff
@@ -41,9 +41,9 @@ function unpackFile() {
 function relativeImagePath(imageFilepath) {
   var pathFragments = imageFilepath;
```

Comment:
Would it be helpful for future programmers to add a doc comment about the slash issue? E.g.
```
// Because we use node-glob, we must use `/` as a separator on all platforms. See https://github.com/isaacs/node-glob#windows
```

Reply:
Fixed, please check again :)

Reply:
That's a really elegant way of doing it. 👍